DigitalLearningSolutions.Data/Models/Register/DelegateRegistrationModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace DigitalLearningSolutions.Data.Models.Register
+{
+    public class DelegateRegistrationModel
+    {
+        public DelegateRegistrationModel(string firstName, string lastName, string email, int centre, int jobGroup,

```

Comment:
Ah, this is minor, but when the lines get so long that we need to split the params onto new lines, we should put each param onto a new line so it's easier to read.